lib/ExtUtils/MM_AIX.pm

```diff
@@ -53,6 +48,21 @@ sub xs_dlsyms_ext {
     '.exp';
 }
 
+sub xs_dlsyms_arg {
+    my($self, $file) = @_;
+    return qq{-bE:${file}};
```

Comment:
This was previously a slightly longer thing, and I kept the highly defensive `${}`.